test/libsolidity/syntaxTests/functionCalls/internal_functions_with_calldata_args_assigned_to_function_pointers_of_same_type.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+    function g(bytes calldata b) pure internal returns (bytes calldata) {
+        return b[2:5];
+    }
```

Comment:
This bit would only be useful if this was a semantic test. In a syntax test we can keep it simple:
```suggestion
    function g(bytes calldata) pure internal returns (bytes calldata) {}
```

Reply:
We need to return a `bytes calldata` variable, so we can't keep it blank.

I'll replace it with a more plain `return b;`

Reply:
Ah true. `b` is fine then.